lang/LangSource/PyrMathOps.cpp

```diff
@@ -675,31 +675,31 @@ int doSpecialBinaryArithMsg(VMGlobals* g, int numArgsPushed, bool isPrimitive) {
                 SetFloat(a, pow((double)slotRawInt(a), (double)slotRawInt(b)));
                 break;
             case opShiftLeft: {
-                long ia = slotRawInt(a);
-                long ib = slotRawInt(b);
+                int64_t ia = slotRawInt(a);
```

Comment:
`auto`?

Reply:
I think maybe the intention is to move to explicit types but ensure generated code doesn't change - in that case, I think this is correct.

Reply:
Again, this is the long as int32 vs long as int64 issue, I just assumed since we are about to do some shifting, that the larger type was needed.